code/controllers/subsystem/timer.dm

```diff
@@ -289,11 +289,20 @@ var/datum/subsystem/timer/SStimer
 	prev = null
 	return QDEL_HINT_QUEUE
 
-proc/addtimer(datum/callback/callback, wait, flags)
+var/list/addtimer_overhead_calls
+
+proc/addtimer(datum/callback/callback, wait, flags, suppress_zero_warning = FALSE)
 	if (!callback)
 		return
 
+
 	if (wait <= 0)
+		if(!suppress_zero_warning)
```

Comment:
Ya, thats what i'm saying, if you are going to change addtimer by making a new system, make this actually useful by having a different behavior rather then just a warning sink that will be mostly triggered from shit doing calculations on wait.

Meaning they would have to add more boilerplate code for no reason.